### DISCUSSION TITLE
PID adjusting

### DISCUSSION THREAD

**Discussion by Coctus** [thorrak/fermentrack#551] [Q&A] [answered]
Any suggestions how to improve my PID adjustments? The PID is now heating and then cooling although the cooling would not be needed in my opinion. Beer temp is set to 19°C and the PID parameters are currently at default. Temperature idle range top setting is 1 so I think it should not start the cooling cycle but I'm new to adjusting PIDs so don't really know where to start from.

![image](https://user-images.githubusercontent.com/45655445/101979371-51ff9580-3c65-11eb-835a-a6aa1c4bf8ed.png)

**Comment by uSlackr** [comment, marked as answer]
There is a convo on this in issue #506.
